sui_programmability/examples/fungible_tokens/sources/regulated_coin.move

```diff
@@ -298,47 +295,10 @@ module abc::tests {
     use abc::abc::{Self, Abc, AbcTreasuryCap, Registry};
     use rc::regulated_coin::{Self as rcoin, RegulatedCoin as RCoin};
 
-    use sui::coin::{Coin};
     use sui::test_scenario::{Self, Scenario, next_tx, ctx};
 
     // === Test handlers; this trick helps reusing scenarios ==
 
-    fun test_minting() {
-        let scenario = scenario();
-        test_minting_(&mut scenario);
-        test_scenario::end(scenario);
-    }
-    fun test_creation() {
-        let scenario = scenario();
-        test_creation_(&mut scenario);
-        test_scenario::end(scenario);
-    }
-    fun test_transfer() {
-        let scenario = scenario();
-        test_transfer_(&mut scenario);
-        test_scenario::end(scenario);
-    }
-    fun test_burn() {
-        let scenario = scenario();
-        test_burn_(&mut scenario);
-        test_scenario::end(scenario);
-    }
-    fun test_take() {
-        let scenario = scenario();
-        test_take_(&mut scenario);
-        test_scenario::end(scenario);
-    }
-    fun test_put_back() {
-        let scenario = scenario();
-        test_put_back_(&mut scenario);
-        test_scenario::end(scenario);
-    }
-    fun test_ban() {
-        let scenario = scenario();
-        test_ban_(&mut scenario);
-        test_scenario::end(scenario);
-    }
```

Comment:
Are these supposed to have `#[test]` on them?

Reply:
I don't know. Hence me tagging @damirka as a reviewer here. I figured we can always bring these back if I was wrong to delete them - they looked to me like (unused) helper functions but I could be wrong of course.